drivers/soundwire/stream.c

```diff
@@ -1389,7 +1389,7 @@ static int _sdw_prepare_stream(struct sdw_stream_runtime *stream,
 
 	ret = do_bank_switch(stream);
 	if (ret < 0) {
-		dev_err(bus->dev, "Bank switch failed: %d\n", ret);
+		dev_err(bus->dev, "do_bank_switch failed: %d\n", ret);
```

Comment:
@plbossart is this even if you have the +pmf flag set for the module?

Reply:
+pmf is for dev_dbg, for dev_err we have unfortunately no means to add more qualifiers.